run/types/DeviceWrapper.ts

```diff
@@ -562,6 +583,26 @@ export class DeviceWrapper implements SharedDeviceInterface {
     return selector;
   }
 
+  public async findText(
+    strategy: Strategy,
+    selector: string,
+    text: string
+  ): Promise<AppiumNextElementType> {
+    let el: null | AppiumNextElementType = null;
+    console.log(
+      `Waiting for accessibility ID '${selector}' to be present with ${text}`
+    );
+    while (el === null) {
+      try {
+        const els = await this.findElements(strategy, selector);
+        el = await this.findMatchingTextInElementArray(els, text);
+      } catch (e) {
+        console.log(`findText threw an error`);
+      }
```

Comment:
should you stop after some time ?

Reply:
Well turns out i'm not using that function anywhere at all... so probably not